templates/mongodb-atlas-peering.template.yaml

```diff
@@ -226,19 +220,25 @@ Resources:
     Type: MongoDB::Atlas::DatabaseUser
     DependsOn: AtlasCluster
     Properties:
-      ProjectId: !Ref "AtlasProject"
-      ApiKeys:
-        PublicKey:  !Ref "PublicKey"
-        PrivateKey: !Ref "PrivateKey"
-      Username: !GetAtt "AtlasIAMRole.Arn"
-      DatabaseName: "$external"
-      AWSIAMType: "ROLE"
+      ProjectId: !GetAtt "AtlasProject.Id"
+      Profile:  !Ref "Profile"
+      Username: !Ref "DatabaseUserName"
+      Password: !Ref "DatabasePassword"
+      DatabaseName:  "admin"
       Roles:
         - RoleName: "readWrite"
           DatabaseName: !Ref "DatabaseUserRoleDatabaseName"
       Scopes:
         - Name: !Sub "${AWS::StackName}"
           Type: "CLUSTER"
+  NetworkContainer:
+    Type: MongoDB::Atlas::NetworkContainer
+    DependsOn: AtlasCluster
+    Properties:
+      Profile:  !Ref "Profile"
+      AtlasCidrBlock: !Ref AtlasCidrBlock
+      RegionName: !Ref ClusterRegion
+      ProjectId: !GetAtt "AtlasProject.Id"
```

Comment:
Don't we need also the [network-peering](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/cfn-resources/network-peering) resource for this template?

Reply:
It seems to me that you are only creating a container without setting up the network peering

Reply:
L177

Reply:
I missed it. Thanks!